README.md

```diff
@@ -43,6 +43,18 @@ If the application will be making many Butter API calls, it is recommended to st
 
 Each Butter client method has a synchronous version and an asynchronous version. Asynchronous methods are appended with the word "Async".
 
+### Preview mode
+
+Preview mode can be used to setup a staging website for previewing content fields or for testing content during local development. To fetch content from preview mode add an additional argument, `true`, to the package initialization:
+
+```C#
+using ButterCMS;
+...
+bool previewMode = true;
+
+var butterClient = new ButterCMSClient("API KEY", previewMode);
```

Comment:
I don't think `butterClient = new ButterCMSClient("API KEY", previewMode);` is accurate now because preview is not the 2nd param. Probably use explicit named params like `preview = true` or something like that.

Reply:
@vlnevyhosteny can you update README for this?

Reply:
Fixed, thanks.